src/common/components/Input.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  forwardRef,
+  useMemo,
+  ChangeEvent,
+  ComponentProps,
+  ReactElement,
+} from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+import classes from './Input.module.scss';
+
+interface InputInterface extends ComponentProps<'input'> {
+  label: ReactElement;
+  errors?: boolean;
+  onChangeAlso?: (event: ChangeEvent<HTMLInputElement>) => void;
+  validated?: boolean;
+}
+
+export const Input = forwardRef<HTMLInputElement, InputInterface>(
+  (props, ref) => {
+    const { className, errors, onChangeAlso, validated, label, ...rest } =
+      props;
+    const { name, onChange } = rest; // react-hook-form internals
+    const idForLabel = useMemo(() => `input-${uuidv4()}`, []);
+    const statusClass = errors ? classes.error : classes.success;
+    const inputClass = [
+      classes.input,
+      validated ? statusClass : '',
+      className,
+    ].join(' ');
+    const labelClass = [
+      classes.label,
+      validated ? statusClass : '',
+      className,
+    ].join(' ');
+    let handler = onChange;
+    if (onChangeAlso && onChange) {
+      handler = (event) => {
+        onChangeAlso(event);
+        onChange(event);
+      };
+    }
```

Comment:
Just in case the onChange is "used" but not truthy, so that onChangeAlso doesn't mysteriously not work, I recommend maybe:
```suggestion
    let handler = onChange || onChangeAlso;
    if (onChangeAlso && onChange) {
      handler = (event) => {
        onChangeAlso(event);
        onChange(event);
      };
    }
```

Reply:
alternatively just:
```suggestion
      handler = (event) => {
        if(onChangeAlso) onChangeAlso(event);
        if(onChange) onChange(event);
      };
```

Reply:
Further, it looks like `register` can take an optional "onChange" that may eliminate our need for onChangeAlso, assuming we don't have other places where onChangeAlso is needed alongside onChange

Reply:
https://react-hook-form.com/api/useform/register

Reply:
Yes, I think that will work for the use case `onChangeAlso` was meaning to address, that is, other side effects of user change events not related to validation. The main difference is that instead of it being a prop to `<Input />` it's a parameter in the register options. But users may still have a question of which fires first, `onChange` or validation, and it appears for now it is validation and then `onChange` as seen [here in the repo](https://github.com/react-hook-form/react-hook-form/blob/6ad4da33dcea2584976aeb9d1f0c1c2bc6803000/src/logic/createFormControl.ts#L647-L681).